test/library/packages/UnitTest/UnitTest.chpl

```diff
@@ -0,0 +1,232 @@
+module UnitTest {
+  use Reflection;
+  /*
+    Assert that a boolean condition is true.  If it is false, prints
+    'assert failed' and halts the program.
+
+    :arg test: the boolean condition
+    :type test: `bool`
+  */
+  pragma "insert line file info"
+  pragma "always propagate line file info"
+  proc assertTrue(test: bool) {
```

Comment:
I didn't notice any tests for `assert{True,False}`.

Reply:
I have added tests for all other assert functions.